src/main/kotlin/ProgramacionOrientadaObjetos/classObjectPropInst25.kt

```diff
@@ -0,0 +1,68 @@
+package ProgramacionOrientadaObjetos
+
+import ProgramacionOrientadaObjetos.clases.Nota
+import java.util.*
+
+// Una "clase" se declara de la siguiente manera: tenemos la palabra reservada class, después el nombre de la clase iniciado con mayúscula (camelCase)
```

Comment:
En éste caso sería CamelCase

Reply:
Ya modifiqué lo que me comentas directamente en el IDE... los cambios se guardan sólos o qué proceso debo realizar antes de hacer "Merge"?
Y cuál de las opciones utilizo?
- Merge pull request
- Create a merge commit
- Squash and merge
- Rebase and merge